doc/level0.rst

```diff
@@ -71,40 +67,48 @@ end-to-end applications, and should be protected from access by other
 applications or co-tenants of the device, at least as well as the
 passwords the MUA retains for the user's IMAP or SMTP accounts.
 
-In Level 0 only one MUA can send and receive encrypted mail through
-Autocrypt mechanisms.  When an Autocrypt-enabled MUA configures an
-e-mail account, it first tries to "claim" the account, to :ref:`lock out
-<lockout>` other MUAs of the same users.  If the claim was
-successful, the MUA should proceed to generate secret keys and store
-them locally.
-
-.. _lockout:
-
-Claiming the Account
---------------------
-
-Only one Level 0 MUA can have Autocrypt enabled for a given account at
-a time.  The Autocrypt-enabled MUA "claims" the account so that others
-disable their Autocrypt features.
-
-The Shared Mail Archive MUST contain a named location mechanism that
-all other Autocrypt clients can see.  For example, an IMAP mailbox
-would have a named folder.  Autocrypt uses the special name
-`_autocrypt_sma` to store "claim" announcements.
-
-The MUA looks in the special location for a message whose form matches
-the standard claim announcement and is valid.  If such a message is
-present, the MUA disables its Autocrypt features for this account.
-
-If the special location does not exist, or it exists, but there are no
-valid claim announcements in it, the MUA crafts its own claim
-announcement and places it in the special location.
-
-.. todo::
-
-   - Document the claim announcement format and precise location
-   - Clarify concerns about race conditions, case-sensitivity, etc.
-
+Avoiding Client Conflicts
+-------------------------
+
+If more than one Autocrypt-enabled client generates a key and then
+distributes it to communication peers, encrypted mail sent to the user
+is only readable by the MUA that sent the last message. This can lead
+to behavior that is unpredictable and confusing for the user.
+
+As a simple measure of mitigation, Level 0 MUAs SHOULD check before
+key generation whether there is evidence in the user's mailbox of
+other active Autocrypt clients. To do this, they SHOULD scan the
+user's Sent folder for mail that contains Autocrypt headers. If such
+mail exists, the MUA SHOULD warn the user and abort key generation,
+unless explicitly instructed to proceed regardless.
+
+In cases where this mechanism cannot be provided, the MUA MUST inform
+the user that they should only set up one Autocrypt client.
+
+To solve this problem in a better way, bi-directional communication
+between the user's different MUAs is required. However, this is out of
+scope for Level 0.
+
+Secret Key Transfer
```

Comment:
I wanted to note down at least a rough roadmap for this, in order to make it easier to pick up by me or someone else who wanted to flesh out the specifics. similar to other TODOs we have in the docs.